docker-compose-cjvp.yml

```diff
@@ -0,0 +1,20 @@
+version: '3.1'
```

Comment:
Pulled this out of the main compose file so we can run as and when needed locally e.g. running the whole suit from one command line operation.